test/notification.js

```diff
@@ -530,35 +530,31 @@ exports.testGlobalOptOut = function(test)
 exports.testMessageWithoutLocalization = function(test)
 {
   let notification = {message: "non-localized"};
-  let texts = Notification.getLocalizedTexts(notification, "en-US");
+  let texts = Notification.getLocalizedTexts(notification);
   test.equal(texts.message, "non-localized");
   test.done();
 };
 
 exports.testLanguageOnly = function(test)
 {
-  let notification = {message: {fr: "fr"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
-  texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr");
+  let notification = {message: {en: "en"}};
+  let texts = Notification.getLocalizedTexts(notification);
+  test.equal(texts.message, "en");
   test.done();
 };
 
 exports.testLanguageAndCountry = function(test)
 {
-  let notification = {message: {"fr": "fr", "fr-CA": "fr-CA"}};
-  let texts = Notification.getLocalizedTexts(notification, "fr-CA");
-  test.equal(texts.message, "fr-CA");
-  texts = Notification.getLocalizedTexts(notification, "fr");
-  test.equal(texts.message, "fr");
+  let notification = {message: {"en": "en", "en-US": "en-US"}};
+  let texts = Notification.getLocalizedTexts(notification);
+  test.equal(texts.message, "en-US");
   test.done();
 };
 
 exports.testMissingTranslation = function(test)
 {
   let notification = {message: {"en-US": "en-US"}};
```

Comment:
Here the idea was that the value of `locale` (which is currently `Utils.appLocale`) does not match the language defined in the `Notification` object, so the default value is used, which is for `en` as far as I remember. So, could you please set `Utils.appLocale` here to something like `"fr"` for example?